tests/integration_tests/saas/test_sentry_task.py

```diff
@@ -205,22 +207,53 @@ def test_sentry_access_request_task(
     )
 
 
+def _get_issues(
+    project: Dict[str, Any],
+    secrets: Dict[str, Any],
+    headers: Dict[str, Any],
+) -> Optional[Dict[str, Any]]:
+    response = requests.get(
+        f"https://{secrets['host']}/api/0/projects/{project['organization']['slug']}/{project['slug']}/issues/",
+        headers=headers,
+    )
+    if not response.ok or response.json() == []:
+        return None
+
+    return response.json()
```

Comment:
In general I'd prefer if a function always returned the same type. This allows the caller to better anticipate the results, and reduces unforeseen errors. Maybe just:
```suggestion
    return response.json() if response.ok else []
```
?

And we allow the caller to always iterate over the returned list? This would be a no-op for empty lists.

Reply:
Fixed the return type for the function, I still need to return None in the event of a non-200 status code or and empty list.

Reply:
If you insist. I still think it can be simplified though:
```suggestion
    json = response.json()
    return json if response.ok and len(json) else None
```

Reply:
This one works, simplifying the expression to this